pallets/pips/src/lib.rs

```diff
@@ -1116,6 +1145,7 @@ impl<T: Trait> Module<T> {
             <ProposalMetadata<T>>::remove(id);
             <Proposals<T>>::remove(id);
             <Referendums<T>>::remove(id);
+            Self::deposit_event(RawEvent::PipPruned(current_did, id));
         }
         Self::deposit_event(RawEvent::PipClosed(current_did, id, prune));
```

Comment:
`PipPruned` event seems redundant since `PipClosed` already has a parameter for it.

Reply:
I went back and forth on this a bit. My "concern" was that for a PIP which was closed and then pruned, you'd have two PipClosed events which seems semantically a bit wrong, so the PipPruned was added to differentiate the two.

Having said that, on reflection it seems cleaner than having two separate events etc. so will remove this extra event.